### PR TITLE
feat: AAU, I see the trust graph of an user

### LINKED ISSUE #14: AAU, I see the trust graph of an user
As a user,
I see the trust graph of an account
So that I see transparently the trusted peers of any account

----

## Technical notes:
* Go with a list similar to the list of snaps assertions, except if using a graph libraries is approximatively the same effort
* If graph representation: Possible Graph libraries (d3js.org, deck.gl, D3.js, Chart.js, Highcharts, Three.js, Vis.js, Plotly.js)
* 

## Out of scope:
* Cover in the user profile header

## Acceptance Criteria:
- [ ] Given that I am navigating in an user profile I see on the header the trust graph with a length of two edges

## Definition of Ready
- [x] Figma design ready
- [x] Acceptance criteria ready
- [ ] Technical grooming done

## Mockup design

=> [FIGMA](https://www.figma.com/file/8N6pTs2HnDJvGQ1tVPTPox/Directory?type=design&node-id=3443%3A65728&mode=design&t=FaF0ch89ETKBlfYz-1)

### PR DESCRIPTION
**Pull request by satyajeetkolhapure**
## Description

First version of profile with connected nodes.
Maximum of two connected nodes are displayed up-to two levels.
Positions of nodes are static.

### Related ticket

Fixes #14 

### Type of change

- [ ] Chore
- [ ] Bug fix
- [x] New feature
- [ ] Documentation update

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,10 +67,12 @@
     "@metamask/snaps-sdk": "^2.0.0",
     "@metamask/utils": "^8.1.0",
     "@reduxjs/toolkit": "^1.9.6",
+    "@types/d3": "^7.4.3",
```
Comment (alainncls) [inline package.json:70]: This line should be in the `devDependencies`

**Comment by satyajeetkolhapure** [comment]
> **🚨 Potential security issues detected.** Learn more about [Socket for GitHub ↗︎](https://socket.dev?utm_medium=gh)
> 
> To accept the risk, merge this PR and you will not be notified again.
> 
> Alert	Package	Note
> [ Network access ](https://socket.dev/npm/package/d3-fetch/issues/3.0.1?issue=networkAccess)	[ npm/d3-fetch@3.0.1 ](https://socket.dev/npm/package/d3-fetch/overview/3.0.1)	
> * **Module:** globalThis["fetch"]
> * **Location:** [Package overview](https://socket.dev/npm/package/d3-fetch/overview/3.0.1)
> 
> [**View full report↗︎**](https://socket.dev/dashboard/org/gh/MetaMask/diff/6b897880-ee72-4d79-a48f-a73a930e0e25/8820c562-8828-403e-9a54-8f4cbf952033)
> 
> ### Next steps
> ** What is network access? **
> ** Take a deeper look at the dependency **
> ** Remove the package **
> **Mark a package as acceptable risk**
> To ignore an alert, reply with a comment starting with `@SocketSecurity ignore` followed by a space separated list of `ecosystem/package-name@version` specifiers. e.g. `@SocketSecurity ignore npm/foo@1.0.0` or ignore all packages with `@SocketSecurity ignore-all`
> 
> * `@SocketSecurity ignore npm/d3-fetch@3.0.1`

@SocketSecurity ignore npm/d3-fetch@3.0.1

File: src/components/ConnectedNodes.test.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-restricted-globals */
```
Comment (alainncls) [inline src/components/ConnectedNodes.test.tsx:1]: Let's avoid this.
I think you should look into `src/types/globals.d.ts`, to add `speechSynthesis` to the Window global

**Comment by satyajeetkolhapure** [comment]
> **New and removed dependencies detected.** Learn more about [Socket for GitHub ↗︎](https://socket.dev?utm_medium=gh)
> 
> Package	New capabilities	Transitives	Size	Publisher
> [npm/@types/d3-array@3.2.1](https://socket.dev/npm/package/@types/d3-array/overview/3.2.1)	None	[`0`](https://socket.dev/npm/package/@types/d3-array/dependencies/3.2.1)	51.4 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-axis@3.0.6](https://socket.dev/npm/package/@types/d3-axis/overview/3.0.6)	None	[`+1`](https://socket.dev/npm/package/@types/d3-axis/dependencies/3.0.6)	81.7 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-brush@3.0.6](https://socket.dev/npm/package/@types/d3-brush/overview/3.0.6)	None	[`+1`](https://socket.dev/npm/package/@types/d3-brush/dependencies/3.0.6)	81.9 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-chord@3.0.6](https://socket.dev/npm/package/@types/d3-chord/overview/3.0.6)	None	[`0`](https://socket.dev/npm/package/@types/d3-chord/dependencies/3.0.6)	25.4 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-color@3.1.3](https://socket.dev/npm/package/@types/d3-color/overview/3.1.3)	None	[`0`](https://socket.dev/npm/package/@types/d3-color/dependencies/3.1.3)	27.9 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-contour@3.0.6](https://socket.dev/npm/package/@types/d3-contour/overview/3.0.6)	None	[`+2`](https://socket.dev/npm/package/@types/d3-contour/dependencies/3.0.6)	73 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-delaunay@6.0.4](https://socket.dev/npm/package/@types/d3-delaunay/overview/6.0.4)	None	[`0`](https://socket.dev/npm/package/@types/d3-delaunay/dependencies/6.0.4)	14.2 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-dispatch@3.0.6](https://socket.dev/npm/package/@types/d3-dispatch/overview/3.0.6)	None	[`0`](https://socket.dev/npm/package/@types/d3-dispatch/dependencies/3.0.6)	5.99 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-drag@3.0.7](https://socket.dev/npm/package/@types/d3-drag/overview/3.0.7)	None	[`+1`](https://socket.dev/npm/package/@types/d3-drag/dependencies/3.0.7)	91.8 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-dsv@3.0.7](https://socket.dev/npm/package/@types/d3-dsv/overview/3.0.7)	None	[`0`](https://socket.dev/npm/package/@types/d3-dsv/dependencies/3.0.7)	28.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-ease@3.0.2](https://socket.dev/npm/package/@types/d3-ease/overview/3.0.2)	None	[`0`](https://socket.dev/npm/package/@types/d3-ease/dependencies/3.0.2)	13.4 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-fetch@3.0.7](https://socket.dev/npm/package/@types/d3-fetch/overview/3.0.7)	None	[`+1`](https://socket.dev/npm/package/@types/d3-fetch/dependencies/3.0.7)	45.5 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-force@3.0.9](https://socket.dev/npm/package/@types/d3-force/overview/3.0.9)	None	[`0`](https://socket.dev/npm/package/@types/d3-force/dependencies/3.0.9)	50.8 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-format@3.0.4](https://socket.dev/npm/package/@types/d3-format/overview/3.0.4)	None	[`0`](https://socket.dev/npm/package/@types/d3-format/dependencies/3.0.4)	18 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-geo@3.1.0](https://socket.dev/npm/package/@types/d3-geo/overview/3.1.0)	None	[`+1`](https://socket.dev/npm/package/@types/d3-geo/dependencies/3.1.0)	73.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-hierarchy@3.1.6](https://socket.dev/npm/package/@types/d3-hierarchy/overview/3.1.6)	None	[`0`](https://socket.dev/npm/package/@types/d3-hierarchy/dependencies/3.1.6)	39.3 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-interpolate@3.0.4](https://socket.dev/npm/package/@types/d3-interpolate/overview/3.0.4)	None	[`+1`](https://socket.dev/npm/package/@types/d3-interpolate/dependencies/3.0.4)	52.4 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-path@3.1.0](https://socket.dev/npm/package/@types/d3-path/overview/3.1.0)	None	[`0`](https://socket.dev/npm/package/@types/d3-path/dependencies/3.1.0)	8.3 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-polygon@3.0.2](https://socket.dev/npm/package/@types/d3-polygon/overview/3.0.2)	None	[`0`](https://socket.dev/npm/package/@types/d3-polygon/dependencies/3.0.2)	6.5 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-quadtree@3.0.6](https://socket.dev/npm/package/@types/d3-quadtree/overview/3.0.6)	None	[`0`](https://socket.dev/npm/package/@types/d3-quadtree/dependencies/3.0.6)	11.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-random@3.0.3](https://socket.dev/npm/package/@types/d3-random/overview/3.0.3)	None	[`0`](https://socket.dev/npm/package/@types/d3-random/dependencies/3.0.3)	14.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-scale-chromatic@3.0.3](https://socket.dev/npm/package/@types/d3-scale-chromatic/overview/3.0.3)	None	[`0`](https://socket.dev/npm/package/@types/d3-scale-chromatic/dependencies/3.0.3)	27.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-scale@4.0.8](https://socket.dev/npm/package/@types/d3-scale/overview/4.0.8)	None	[`+1`](https://socket.dev/npm/package/@types/d3-scale/dependencies/4.0.8)	166 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-selection@3.0.10](https://socket.dev/npm/package/@types/d3-selection/overview/3.0.10)	None	[`0`](https://socket.dev/npm/package/@types/d3-selection/dependencies/3.0.10)	66 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-shape@3.1.6](https://socket.dev/npm/package/@types/d3-shape/overview/3.1.6)	None	[`+1`](https://socket.dev/npm/package/@types/d3-shape/dependencies/3.1.6)	145 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-time-format@4.0.3](https://socket.dev/npm/package/@types/d3-time-format/overview/4.0.3)	None	[`0`](https://socket.dev/npm/package/@types/d3-time-format/dependencies/4.0.3)	12.6 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-time@3.0.3](https://socket.dev/npm/package/@types/d3-time/overview/3.0.3)	None	[`0`](https://socket.dev/npm/package/@types/d3-time/dependencies/3.0.3)	30.7 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-timer@3.0.2](https://socket.dev/npm/package/@types/d3-timer/overview/3.0.2)	None	[`0`](https://socket.dev/npm/package/@types/d3-timer/dependencies/3.0.2)	6.75 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-transition@3.0.8](https://socket.dev/npm/package/@types/d3-transition/overview/3.0.8)	None	[`+1`](https://socket.dev/npm/package/@types/d3-transition/dependencies/3.0.8)	108 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3-zoom@3.0.8](https://socket.dev/npm/package/@types/d3-zoom/overview/3.0.8)	None	[`+3`](https://socket.dev/npm/package/@types/d3-zoom/dependencies/3.0.8)	155 kB	[types](https://socket.dev/npm/user/types)
> [npm/@types/d3@7.4.3](https://socket.dev/npm/package/@types/d3/overview/7.4.3)	None	[`+31`](https://socket.dev/npm/package/@types/d3/dependencies/7.4.3)	1 MB	[types](https://socket.dev/npm/user/types)
> [npm/@types/geojson@7946.0.14](https://socket.dev/npm/package/@types/geojson/overview/7946.0.14)	None	[`0`](https://socket.dev/npm/package/@types/geojson/dependencies/7946.0.14)	8.11 kB	[types](https://socket.dev/npm/user/types)
> [npm/d3-array@3.2.4](https://socket.dev/npm/package/d3-array/overview/3.2.4)	None	[`+1`](https://socket.dev)	174 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-axis@3.0.0](https://socket.dev/npm/package/d3-axis/overview/3.0.0)	None	[`0`](https://socket.dev)	31.4 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-brush@3.0.0](https://socket.dev/npm/package/d3-brush/overview/3.0.0)	None	[`+8`](https://socket.dev)	554 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-chord@3.0.1](https://socket.dev/npm/package/d3-chord/overview/3.0.1)	None	[`+1`](https://socket.dev)	59.4 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-color@3.1.0](https://socket.dev/npm/package/d3-color/overview/3.1.0)	None	[`0`](https://socket.dev)	61.2 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-contour@4.0.2](https://socket.dev/npm/package/d3-contour/overview/4.0.2)	None	[`+2`](https://socket.dev)	224 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-delaunay@6.0.4](https://socket.dev/npm/package/d3-delaunay/overview/6.0.4)	None	[`+2`](https://socket.dev)	456 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-dispatch@3.0.1](https://socket.dev/npm/package/d3-dispatch/overview/3.0.1)	None	[`0`](https://socket.dev)	15.6 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-drag@3.0.0](https://socket.dev/npm/package/d3-drag/overview/3.0.0)	None	[`+2`](https://socket.dev)	193 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-dsv@3.0.1](https://socket.dev/npm/package/d3-dsv/overview/3.0.1)	Transitive: [filesystem](https://socket.dev?tab=dependencies&issue=filesystemAccess)	[`+3`](https://socket.dev)	466 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-ease@3.0.1](https://socket.dev/npm/package/d3-ease/overview/3.0.1)	None	[`0`](https://socket.dev)	34.6 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-fetch@3.0.1](https://socket.dev/npm/package/d3-fetch/overview/3.0.1)	[network](https://socket.dev?issue=networkAccess) Transitive: [filesystem](https://socket.dev?tab=dependencies&issue=filesystemAccess)	[`+4`](https://socket.dev)	483 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-force@3.0.0](https://socket.dev/npm/package/d3-force/overview/3.0.0)	None	[`+3`](https://socket.dev)	167 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-format@3.1.0](https://socket.dev/npm/package/d3-format/overview/3.1.0)	None	[`0`](https://socket.dev)	57.8 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-geo@3.1.1](https://socket.dev/npm/package/d3-geo/overview/3.1.1)	None	[`+2`](https://socket.dev)	401 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-hierarchy@3.1.2](https://socket.dev/npm/package/d3-hierarchy/overview/3.1.2)	None	[`0`](https://socket.dev)	136 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-interpolate@3.0.1](https://socket.dev/npm/package/d3-interpolate/overview/3.0.1)	None	[`+1`](https://socket.dev)	131 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-path@3.1.0](https://socket.dev/npm/package/d3-path/overview/3.1.0)	None	[`0`](https://socket.dev)	20.9 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-polygon@3.0.1](https://socket.dev/npm/package/d3-polygon/overview/3.0.1)	None	[`0`](https://socket.dev)	14.8 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-quadtree@3.0.1](https://socket.dev/npm/package/d3-quadtree/overview/3.0.1)	None	[`0`](https://socket.dev)	43.4 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-random@3.0.1](https://socket.dev/npm/package/d3-random/overview/3.0.1)	None	[`0`](https://socket.dev)	38.1 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-scale-chromatic@3.1.0](https://socket.dev/npm/package/d3-scale-chromatic/overview/3.1.0)	None	[`+2`](https://socket.dev)	207 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-scale@4.0.2](https://socket.dev/npm/package/d3-scale/overview/4.0.2)	None	[`+7`](https://socket.dev)	687 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-selection@3.0.0](https://socket.dev/npm/package/d3-selection/overview/3.0.0)	None	[`0`](https://socket.dev)	135 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-shape@3.2.0](https://socket.dev/npm/package/d3-shape/overview/3.2.0)	None	[`+1`](https://socket.dev)	268 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-time-format@4.1.0](https://socket.dev/npm/package/d3-time-format/overview/4.1.0)	None	[`+3`](https://socket.dev)	324 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-time@3.1.0](https://socket.dev/npm/package/d3-time/overview/3.1.0)	None	[`+2`](https://socket.dev)	239 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-timer@3.0.1](https://socket.dev/npm/package/d3-timer/overview/3.0.1)	None	[`0`](https://socket.dev)	18.4 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-transition@3.0.1](https://socket.dev/npm/package/d3-transition/overview/3.0.1)	None	[`+6`](https://socket.dev)	445 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3-zoom@3.0.0](https://socket.dev/npm/package/d3-zoom/overview/3.0.0)	None	[`+8`](https://socket.dev)	575 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/d3@7.9.0](https://socket.dev/npm/package/d3/overview/7.9.0)	Transitive: [filesystem](https://socket.dev?tab=dependencies&issue=filesystemAccess), [network](https://socket.dev?tab=dependencies&issue=networkAccess)	[`+36`](https://socket.dev)	4.02 MB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/delaunator@5.0.1](https://socket.dev/npm/package/delaunator/overview/5.0.1)	None	[`+1`](https://socket.dev)	351 kB	[mourner](https://socket.dev/npm/user/mourner)
> [npm/internmap@2.0.3](https://socket.dev/npm/package/internmap/overview/2.0.3)	None	[`0`](https://socket.dev)	10.5 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/robust-predicates@3.0.2](https://socket.dev/npm/package/robust-predicates/overview/3.0.2)	None	[`0`](https://socket.dev)	296 kB	[mourner](https://socket.dev/npm/user/mourner)
> [npm/rw@1.3.3](https://socket.dev/npm/package/rw/overview/1.3.3)	[filesystem](https://socket.dev?issue=filesystemAccess)	[`0`](https://socket.dev)	24.2 kB	[mbostock](https://socket.dev/npm/user/mbostock)
> [npm/zustand@4.5.2](https://socket.dev/npm/package/zustand/overview/4.5.2)	[environment](https://socket.dev?issue=envVars)	[`+9`](https://socket.dev)	3.26 MB	[daishi](https://socket.dev/npm/user/daishi)
> **🚮 Removed packages:** [npm/strip-eof@1.0.0](https://socket.dev/npm/package/strip-eof/overview/1.0.0)
> 
> [**View full report↗︎**](https://socket.dev/dashboard/org/gh/MetaMask/diff/ed79abcc-06d7-4225-a1b2-ebf071192f4b/48d0abb3-190d-4832-ba31-f0504abbdf4e)

@SocketSecurity ignore npm/d3-fetch@3.0.1

**Review by alainncls — CHANGES_REQUESTED** [review]

**Comment by satyajeetkolhapure** [comment]
@SocketSecurity ignore npm/d3-fetch@3.0.1

**Pull request merged**